utils/runtime-fuzzer/fuzz/fuzz_targets/main.rs

```diff
@@ -44,21 +44,31 @@ fuzz_target!(|seed: u64| {
 // directory before running fuzz test.
 //
 // If directory already exists for the current run, it will be cleared.
-fn dump_seed(seed: u64) -> Result<(), String> {
+fn dump_seed(seed: u64) -> IoResult<()> {
     let seeds_file = RUN_INTIALIZED.get_or_try_init(|| {
         let seeds_dir = Path::new(SEEDS_STORE_DIR);
-        if seeds_dir.exists() {
-            fs::remove_dir_all(seeds_dir).map_err(|e| e.to_string())?;
+        match seeds_dir.exists() {
+            true => clear_dir(seeds_dir)?,
+            false => fs::create_dir_all(seeds_dir)?,
         }
```

Comment:
`match` for `bool` is overkill in my opinion. I'd use the old good `if`-`else`.

Reply:
`match` is shorter

Reply:
discussed.